packages/terra-date-picker/src/DateUtil.js

```diff
@@ -83,6 +83,17 @@ class DateUtil {
 
     return undefined;
   }
+
+  /**
+   * Determines if a provided date input value is valid.
+   * Valid inputs are either empty strings or contain only numeric and the `/` character.
```

Comment:
Should also call out the `.`

Reply:
Fixed https://github.com/cerner/terra-framework/pull/595/commits/9af826c35a73bb5fad6c3023e10fbe23063035a5